packages/dds/tree/src/feature-libraries/schemaSummarizer.ts

```diff
@@ -170,6 +170,15 @@ export class SchemaEditor<TRepository extends StoredSchemaRepository>
 		return false;
 	}
 
```

Comment:
Given that this method is a hack to work with out current improper schema handleing, it should be documentd as such (also it should be document in general):

```suggestion

	/**
	 * @returns true iff this is a schema op and was submitted.
	 *
	 * TODO: Shared tree needs a pattern for handling non-changeset operations.
	 * See TODO on `SharedTree.processCore`.
	 */
```

Reply:
added suggested documentation